modules/logger.lua

```diff
@@ -138,6 +138,7 @@ end
 ---@field webhook? string Discord logs only. url of the webhook this log should send to
 ---@field color? string Discord logs only. what color the message should be
 ---@field tags? string[] Discord logs only. tags in discord. Example: {'<@%roleid>', '@everyone'}
+---@field oxLibTags? string[] -- Tags for ox_lib logger
```

Comment:
```suggestion
---@field oxLibTags? string -- Tags for ox_lib logger
```

Testing this it looks like it needs to be just a flat string with : and , to separate kvps. Passing a table just gives table 0x<whatever> in the log. Unless you have an example of how this can be passed as a table.

Reply:
@real-general-iroh do you have an example of this working with a table?